packages/gestalt/src/Tabs.js

```diff
@@ -25,54 +52,68 @@ export default function Tabs({
     text: React.Node,
   |}>,
   wrap?: boolean,
-|}): React.Node {
-  const handleTabClick = (i: number, e: SyntheticMouseEvent<>) =>
-    onChange({ activeTabIndex: i, event: e });
+|};
 
+export default function Tabs({
+  activeTabIndex,
+  onChange,
+  size = 'md',
+  tabs,
+  wrap,
+}: Props): React.Node {
   return (
-    <div
-      className={classnames(
-        styles.Tabs,
-        wrap && layout.flexWrap,
-        size === 'md' ? layout.medium : layout.large
-      )}
+    <Box
+      alignItems="center"
+      display="flex"
+      marginEnd={-1}
+      marginStart={-1}
+      minHeight={size === 'lg' ? 48 : 40}
       role="tablist"
+      wrap={wrap}
+      dangerouslySetInlineStyle={{
+        __style: { userSelect: 'none' },
+      }}
     >
-      {tabs.map(({ href, id, text }, i) => {
-        const isActive = i === activeTabIndex;
-        const cs = classnames(styles.tab, {
-          [styles.tabIsNotActive]: !isActive,
-          [styles.tabIsActive]: isActive,
-        });
+      {tabs.map(({ id, href, text }, index) => {
+        const isActive = activeTabIndex === index;
         return (
-          <a
-            aria-selected={isActive}
-            className={cs}
-            href={href}
-            {...(id ? { id } : {})}
-            key={`${i}${href}`}
-            onClick={(e: SyntheticMouseEvent<>) => handleTabClick(i, e)}
-            role="tab"
+          <Box
+            key={id || `${href}_${index}`}
+            id={id}
```

Comment:
@jackhsu978 what was the reason behind this change? Why did we not leave the `id` on the `anchor` tag instead?

Reply:
because `Link` doesn't support `id` whereas `Box` supports it. I've added the support the prop `dangerouslySetId` to `Link` component, so now the id is associated with the anchor tag.

I'm not sure if we want to add `dangerouslySetId` to `Link` component. I am thinking maybe it's safer to make it less discoverable because it seems dangerous. Any thoughts on this?